src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/types';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
```

Comment:
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
```

if u change TodoInfo user section to check if user exists, ofc.
because if not, then ur component will always render an empty <a class='UserInfo'> tag if there's no user